internal/sdkv2provider/resource_cloudflare_access_application_test.go

```diff
@@ -339,7 +340,7 @@ func TestAccCloudflareAccessApplication_WithSCIMConfigOAuth2(t *testing.T) {
 					resource.TestCheckResourceAttr(name, "scim_config.0.deactivate_on_delete", "true"),
 					resource.TestCheckResourceAttr(name, "scim_config.0.authentication.0.scheme", "oauth2"),
 					resource.TestCheckResourceAttr(name, "scim_config.0.authentication.0.client_id", "beepboop"),
-					resource.TestCheckResourceAttrSet(name, "scim_config.0.authentication.0.client_secret"),
+					//resource.TestCheckResourceAttrSet(name, "scim_config.0.authentication.0.client_secret"),
```

Comment:
what are we doing here? should this be fixed now?

Reply:
yeah, these tests should be fixed now